script.js

```diff
@@ -0,0 +1,89 @@
+$(document).ready(function(){
+    var mainBoard = new Board($("#board"));
+
+});
+
+function Board(element) {
+    //this.element == mainBoard which is my entire container div
+    var self = this;
+    this.element = element;
+    this.numOfMoves = 0;
+
+
+    $(".box").on("click", function(){
+        //$(this) is referring to .box ie whatever the listener is listening for.
+        // console.log($(this));
+        countMoves();
+        addXorO($(this));
+    });
+
+    $("#reset").on("click", function(){
+        $(".box").each(function(){
+            $(this).html("");
+        });
+    });
+
+    function countMoves() {
+        console.log("You clicked me!");
+        //add 1 to moves.
+        self.numOfMoves++;
+    }
+
+    //switching between x's and o's
+    function addXorO(square){
+        if(self.numOfMoves % 2 === 0){
+            //get inner html and add an O
+            square.html("O");
+        }else{
+            //enter an X
+            square.html("X");
+        }
+        declareWinner();
+    }
```

Comment:
As your game currently stands, I can click on a square that already contains an `X` or `O` and replace it with the opposite character. How could you use a conditional(s) to prevent this from happening?